project/Dependencies.scala

```diff
@@ -118,6 +117,8 @@ object Dependencies {
 
       val levelDB = "org.iq80.leveldb" % "leveldb" % "0.10" % "optional;provided" // ApacheV2
       val levelDBNative = "org.fusesource.leveldbjni" % "leveldbjni-all" % "1.8" % "optional;provided" // New BSD
+
+      val junit = Compile.junit % "optional;provided;test"
```

Comment:
good to have it provided, remember to add to reference docs that the dependency must be added when using the `TestKitJunitResource`